modules/updating-eus-to-eus-upgrade-cli.adoc

```diff
@@ -0,0 +1,150 @@
+// Module included in the following assemblies:
+//
+// * updating/preparing-eus-eus-upgrade.adoc
+
+:_content-type: PROCEDURE
+[id="updating-eus-to-eus-upgrade-cli_{context}"]
+= EUS-to-EUS upgrade using the CLI
+
+.Prerequisites
+
+* Upgrade OpenShift CLI `oc` to the target version before each upgrade.
+
+[IMPORTANT]
+====
+If the OpenShift CLI `oc` is not upgraded to the target version before your upgrade, the `oc adm upgrade` channel does not retry properly.
```

Comment:
@obrown1205 sorry if I may mislead you by mentioning that bug in the previous review. my intention never was to include its details, but rather to highlight the importance of keeping the oc client up to date while upgrading a cluster. in my opinion, we may want to provide a more general kind of mention, for example: "potentially create unexpected issues" or "may render the cluster unstable" or better yet "is unsupported and highly discouraged". however is not a request, but rather a suggestion. other reviewers may suggest otherwise as well.

Reply:
Ah :) That makes sense! Thank you very much for the clarification. I've updated to say "It is highliy discouraged to skip this prerequisite. If the OpenShift CLI `oc` is not upgraded to the target version before your upgrade, unexpected issues may occur." Thoughts?

Reply:
sounds good thanks!